lib/core/msdf-sys/src/lib.rs

```diff
@@ -0,0 +1,163 @@
+extern crate wasm_bindgen;
+#[macro_use] extern crate slice_as_array;
+
+use wasm_bindgen::prelude::*;
+use js_sys::*;
+
+const MAX_MSDF_SIZE : usize = 32;
+pub const MSDF_CHANNELS_COUNT : usize = 3;
```

Comment:
These constants are not described - why 3 and not 7? Any explanation should be provided. Moreover, what does `max msdf size` mean? `msdf` is a name of the library, so is this size of this library in megabytes or something else?

Reply:
MSDF is a Multi-channel signed distance field - a data structure like image. Library name is msdfgen (because it generates msdf). Shall I give here explanation what MSDF is?

Reply:
Yep, a short info would be desirable + link to more info.